app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -4,7 +4,7 @@
             <h2>If you have a degree</h2>
 
             <p>
-                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options you can choose.
+                Find out how to get QTS if you have a degree or you’re studying for one by choosing your postgraduate teacher training course.
```

Comment:
I don't think this quite makes sense now @EllieNodder - sounds like you're studying for a degree **by** choosing a PG course?


```suggestion
                Find out how to get QTS if you have a degree or you’re studying for one. Get tips on how to choose your postgraduate teacher training course.
```